config/crd/kustomization.yaml

```diff
@@ -2,17 +2,20 @@
 # since it depends on service name and namespace that are out of this kustomize package.
 # It should be run by config/default
 resources:
+- bases/misf.ibm.com_managedfusionofferings.yaml
 # +kubebuilder:scaffold:crdkustomizeresource
 
 patchesStrategicMerge:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
 # patches here are for enabling the conversion webhook for each CRD
 #- patches/webhook_in_managedocs.yaml
```

Comment:
maybe as a separate PR, it's better to remove traces of `managedocs` CRDs.

Reply:
Yes, I have noted this down. I will update the files that have managedocs traces.